packages/utilities/exposure-scanning/patch/index.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable global-require */
+let octokit;
+
+// Used to limit the number of calls to the patch comment and review comment endpoint
+const maxCommentPatches = 20;
+
+(async () => {
+  if (process.env.GITHUB_ACTION && process.env.GITHUB_TOKEN) {
+    const { Octokit } = await import('@octokit/action');
+    octokit = new Octokit();
+  } else {
+    const { Octokit } = await import('@octokit/rest');
+    octokit = new Octokit();
+  }
+})();
+
+const patchPrBody = async (reqBody, body) => {
+  await octokit.request('PATCH /repos/{owner}/{repo}/pulls/{id}', {
+    ...reqBody,
+    body,
+  });
+};
+
+const patchReviewComments = async (reqBody, comments) => {
+  await Promise.all(
+    comments.slice(0, maxCommentPatches).map(({ id, body }) =>
+      octokit.request('PATCH /repos/{owner}/{repo}/pulls/comments/{id}', {
+        owner: reqBody.owner,
+        repo: reqBody.repo,
+        id,
+        body,
+      }),
+    ),
+  );
+};
+
+const patchComments = async (reqBody, comments) => {
+  await Promise.all(
+    comments.slice(0, maxCommentPatches).map(({ id, body }) =>
+      octokit.request('PATCH /repos/{owner}/{repo}/issues/comments/{id}', {
+        ...reqBody,
+        id,
+        body,
+      }),
+    ),
+  );
+};
+
+const patchIssueBody = async (reqBody, body) => {
+  await octokit.request('PATCH /repos/{owner}/{repo}/issues/{id}', {
+    ...reqBody,
+    body,
+  });
+};
```

Comment:
```suggestion
const patchIssueBody = (reqBody, body) =>
  octokit.request('PATCH /repos/{owner}/{repo}/issues/{id}', {
    ...reqBody,
    body,
  });
```
Same as above _potentially_